test/unit/spec/common/cellComponents/tabs/infoTab-Spec.js

```diff
@@ -1,109 +1,418 @@
-define(['common/cellComponents/tabs/infoTab', 'testUtil'], (InfoTab, TestUtil) => {
+define(['common/cellComponents/tabs/infoTab', 'common/props', 'testUtil'], (
+    InfoTab,
+    Props,
+    TestUtil
+) => {
     'use strict';
 
     describe('The App Info Tab module', () => {
-        afterEach(() => {
-            TestUtil.clearRuntime();
-        });
-
         it('loads', () => {
             expect(InfoTab).not.toBe(null);
         });
 
         it('has expected functions', () => {
-            expect(InfoTab.make).toBeDefined();
+            expect(InfoTab.make).toEqual(jasmine.any(Function));
+            expect(InfoTab.cssBaseClass).toEqual(jasmine.any(String));
         });
     });
 
     describe('The App Info Tab instance', () => {
-        const model = {
-            getItem: (item) => model[item],
-            app: { tag: 'Mock App' },
-            'app.spec': {
-                full_info: {
-                    description: 'Details about this mock app',
+        afterEach(() => {
+            TestUtil.clearRuntime();
+        });
+
+        const { cssBaseClass } = InfoTab;
+
+        const APP = {
+                ONE: {
+                    ID: 'my_favourite_app/ever',
+                    NAME: 'My Favourite App Ever!!',
+                    TAG: 'beta',
+                    VERSION: '1.0.1',
                 },
-                info: {
-                    authors: ['Abraham', 'Martin', 'John'],
-                    id: 0,
-                    subtitle: 'A mock app for testing purposes',
-                    ver: '1.0.1',
+                TWO: {
+                    ID: 'crap/app',
+                    NAME: 'Crap App',
+                    TAG: 'whatever!',
+                    VERSION: '0.5-beta-6+3',
+                },
+                THREE: {
+                    ID: 'blah-blah-blah',
+                    NAME: 'blah blah blah',
+                    TAG: 'blah',
+                    VERSION: 'halb',
                 },
-                parameters: [
-                    {
-                        text_options: {
-                            valid_ws_types: ['KBaseRNASeq.RNASeqSampleSet'],
-                        },
-                        ui_name: 'RNA sequence object <font color=red>*</font>',
-                    },
-                    {
-                        ui_name: 'Adapters',
-                    },
-                ],
             },
+            DOCSTRING = 'View full documentation',
+            FONT_TAG_CONTENTS = 'Some text in a font tag';
+
+        const appData = {},
+            appModels = {};
+
+        appData[APP.ONE.ID] = {
```

Comment:
I've set up three fake apps to test the various elements of the info tab, both in the situation where the info tab is part of the bulk import cell and when it is its own tab.